docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -211,7 +210,22 @@ If multiple entries are specified in the `host_user_uid` or `host_user_gid` only
 
 </Admonition>
 
-## Step 3/3 Test host user creation
+## Step 3/4 Sudoers Configuration
```

Comment:
```suggestion
## Step 3/4. Configure sudoers on your Linux servers
```
Suggested so the sentence structures of numbered headings are parallel to one another. Otherwise, they are all in the imperative mood except for this one.